servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/RunnableTerminalSignalConsumer.java

```diff
@@ -17,11 +17,17 @@
 
 import static java.util.Objects.requireNonNull;
 
-final class RunnableTerminalSignalConsumer implements TerminalSignalConsumer {
-
+/**
+ * A {@link TerminalSignalConsumer} where each method executes a {@link Runnable#run()}.
+ */
+public final class RunnableTerminalSignalConsumer implements TerminalSignalConsumer {
```

Comment:
If we are ok to expose it as public API, alternative could be to add a static method `TerminalSignalConsumer.from(Runnable)`. That will simplify discoverability and be consistent with other interfaces, like `HostAndPort`, factories, initializers, etc.

Reply:
done